src/attesters/sev/init.c

```diff
@@ -0,0 +1,19 @@
+/* Copyright (c) 2022 Intel Corporation
+ * Copyright (c) 2020-2022 Alibaba Cloud
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <rats-tls/log.h>
+#include <rats-tls/attester.h>
+
+static unsigned int dummy_private;
+
+enclave_attester_err_t sev_attester_init(enclave_attester_ctx_t *ctx, rats_tls_cert_algo_t algo)
+{
+	RTLS_DEBUG("ctx %p, algo %d\n", ctx, algo);
+
+	ctx->attester_private = &dummy_private;
+
```

Comment:
This static variable is initialized may be more better even it is not used in current scenario.